plugins/woocommerce/src/Internal/Admin/Orders/PageController.php

```diff
@@ -143,7 +143,7 @@ public function setup(): void {
 		}
 
 		// Not on an Orders page.
-		if ( 'admin.php' !== $pagenow || 0 !== strpos( $plugin_page, 'wc-orders' ) ) {
+		if ( ! isset( $plugin_page ) || 'admin.php' !== $pagenow || 0 !== strpos( $plugin_page, 'wc-orders' ) ) {
```

Comment:
```suggestion
		if ( empty($plugin_page) || 'admin.php' !== $pagenow || 0 !== strpos( $plugin_page, 'wc-orders' ) ) {
```

A minor optimization, but I'm suggesting an `empty` check instead because that will cover when $plugin_page is both `null` or an empty string which would short circuit the other checks.

Reply:
if you don't mind making the commit, please do.  I'm no php dev, just took the change OP had.  I assume your suggestion makes sense :)

Reply:
whoops, didn't realize github had a feature where i can just press commit.  Very convenient